src/components/GoodList.tsx

```diff
@@ -0,0 +1,15 @@
+import { Good } from '../types/Good.type';
+
+type Props = {
+  goods: Good[];
+};
+
+export const GoodList: React.FC<Props> = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <li key={good} data-cy="Good">
```

Comment:
[CODE KNOWLEDGE] - The `key` prop in a list should be a unique identifier, not the item itself. Use a unique attribute of `good`, such as `good.id`, as the key.